### ISSUE TITLE
Need help

### ISSUE THREAD

**Issue by Ciao121** [ItsVoid/agario-client#4]
Hi,
I'm looking for help about the showOverlays functions.

This is where the function is fired:

```
case 27: // quit
    showOverlays(true);
    wHandle.isSpectating = false;
    break;
```

So when an user press ESC the function is fired and the overlay is shown. But the Overlay is also fired when an user dies. I cannot find where, in this case, the function is fired. I'd like to have a slightly different function to be fired in that case.

Can anyone help?
Thank you!!!

**Comment by Ciao121** [comment]
I try to better explain my issue:
The client's overlay is shown in two cases:
A) When a player presses esc
B) When a player dies

I think the A case is fired in main_out.js in those line
`
 case 27: // quit
     showOverlays(true); 
     wHandle.isSpectating = false;
     break;
`
I'd like to have case B to fire a custom function "showOverlays2(true)" instad of  "showOverlays(true)".
But I really cannot find a way to do this.
Thank you

**Comment by Barbosik** [comment]
Ciao121: look for `$("#stats").show();` - it's what you're looking for

**Comment by ZfsrGhS953** [comment]
@Ciao121 do you want to run 

``` javascript
wHandle.isSpectating = false;
```

 ? Because in https://github.com/ItsVoid/agario-client/blob/master/main_out.js#L74 there isn't a line like that.

**Comment by Ciao121** [comment]
@ZfsrGhS953 you are right, it's not in the original code. But also deleting it or changing it to "true" does not changes.
@Barbosik I can't find that code anywhere...

**Comment by ZfsrGhS953** [comment]
It's

``` javascript
ua && 0 == playerCells.length && showOverlays(false)
```

It's different because when you call that function with false it also shows ads with the menu. Unless you removed them of course, I don't know because i have adblock installed.

**Comment by Ciao121** [comment]
@ZfsrGhS953 thanks. That's solved my issue.
